front/lib/api/assistant/globalAgents.ts

```diff
@@ -0,0 +1,137 @@
+import { Authenticator } from "@app/lib/auth";
+import { DataSource } from "@app/lib/models";
+import { AgentConfigurationType } from "@app/types/assistant/agent";
+
+/**
+ * GLOBAL AGENTS CONFIGURATION
+ *
+ * To add an agent:
+ * - Add a unique SID in GLOBAL_AGENTS_SID.
+ * - Add a unique ID in GLOBAL_AGENTS_ID.
+ * - Add a case in getGlobalAgent with associated function.
+ */
+
+enum GLOBAL_AGENTS_SID {
+  GPT4 = "gpt-4",
+  Slack = "slack",
+}
+enum GLOBAL_AGENTS_ID {
+  GPT4 = -1,
+  Slack = -2,
+}
+
+async function _getGPT4GlobalAgent(): Promise<AgentConfigurationType> {
+  return {
+    id: GLOBAL_AGENTS_ID.GPT4,
+    sId: GLOBAL_AGENTS_SID.GPT4,
+    name: "GPT4",
+    pictureUrl: null,
+    status: "active",
+    scope: "global",
+    generation: {
```

Comment:
Global agent is as well here maybe?

Reply:
You use 0 but we could use the magic negative ids here as well?

Reply:
Yes indeed, thanks! I added a commit 👍🏻